src/puppetlabs/puppetdb/jdbc.clj

```diff
@@ -298,38 +300,30 @@
 
   The cursor is closed when `func` returns. If an exception is thrown,
   the query is cancelled."
-  [func sql params]
+  [[sql & params] func]
```

Comment:
ah, there were two overloads before; the one that takes the vector was actually unused.